### DISCUSSION TITLE
Is there a way to allocate extra outposts to the national capital?

### DISCUSSION THREAD

**Discussion by Bao4555** [TownyAdvanced/Towny#6468] [Q&A] [answered]
I'd like to assign outposts to villages affiliated with the country and additional outposts to the national capital, is there any way I can do it with the system?

I would like to know the method except limit_outposts_using_town_and_nation_levels: 'true'

**Comment by LlmDl** [comment, marked as answer]
The `nationBonusOutpostLimit` values in the config.yml's nation_level adds outposts to towns who are in a nation. There is not a way to give the capital city another additional bonus amount.

**Reply by Bao4555** [reply]
Then, do you have any thoughts of creating a new code?
Or do I have to request it from Discord separately?

**Reply by LlmDl** [reply]
If you have requests you can make a suggestion ticket on github, if you want it to be considered you should become a sponsor.
